### ISSUE TITLE
Grid: grid element is set to editable (4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#943]
### Expected behavior
Grids in SKY UX are for displaying data. They aren't an interactive grid to enter data and should have this property set to false. 

https://www.w3.org/TR/wai-aria/roles#grid
A grid is considered editable unless otherwise specified. To make a grid read-only, set the aria-readonly attribute of the grid to true.

### Actual behavior
aria-readonly is not set on grid element and by default this is set to true.
